handbook/information-security/supplier-policy.md

```diff
@@ -0,0 +1,115 @@
+# Supplier Policy
+
+In providing our services we rely on a number of third party suppliers that have
+access to some of the sensitive information we handle. This policy sets out how
+we vet our suppliers to ensure that we understand and assess the risks involved.
+
+## Objectives
+
+The objectives of our Supplier Policy are to:
+* understand which of our suppliers are handling sensitive information on our
+  behalf
+* understand the risks created by our collaboration
+* ensure the relevant suppliers follow best practices to minimise information
+  security risks
+* work with our suppliers to implement best practice where it is not being
+  implemented
+* make a conscious decision whether the risks we expose ourselves to are
+  acceptable
+* where we are exposed to unacceptable risk, mitigate this by ending the
+  collaboration if necessary
+
+## Vetting Process
+The below diagram illustrates the process that the rest of this document
+describes in more detail.
+
+![Supplier Policy Flowchart](../../assets/supplier-policy.png)
+
+### Supplier List
+
+Our [Supplier
+List](https://docs.google.com/spreadsheets/d/14kog_nFut3v2t5mZNNatZ-oWb7tP3ImWdXjUP9EkZGc/edit?usp=sharing)
+tracks the suppliers we interact with.
+
+This is checked for completeness monthly by the [Information Security
+Officer](../README.md#contacts).
+
+We define suppliers as any organisation that has invoiced us. Therefore the
+monthly update requires a review of our received invoices for new suppliers.
+
+### Supplier Ownership
+
+The Information Security Officer will assign a Supplier Owner to each Supplier
+in the Supplier List. These owners are tracked in the Supplier List. They are
+responsible for conducting the Vetting Process laid out below.
+
+### Vetting Process
+
+#### Vetting Requirement
+
+While we work with a large number of suppliers, not all of them will have access
```

Comment:
This is a bit wordy. I would probably say we organise suppliers into 3 categories. Define the categories along with the associated checks/measures against each supplier.

Reply:
Would the three categories be:
- Large organisation -> review of their publicly available info sec policies
- Medium-sized organisation -> review of their publicly available info sec policies + questionnaire where necessary + negotiation where necessary?
- Individuals and small organisations -> NDA + negotiation?